modules/sdk-core/src/bitgo/utils/mpcUtils.ts

```diff
@@ -10,6 +10,7 @@ import { encryptText, getBitgoGpgPubKey } from './opengpgUtils';
 export interface MpcKeyShare {
   publicShare: string;
   privateShare: string;
+  signature?: string;
```

Comment:
i would change `signature` to be more descriptive.   like `privateShareProof`

(its not a signature, technically)

Reply:
updated